cpg_infra/billing_aggregator/aggregate/seqr.py

```diff
@@ -395,8 +395,16 @@ def generator() -> Generator[dict, None, None]:
 
             json_objs_iter = generator()
 
+    def _append_if_not_present(entries: list[dict], obj: dict) -> None:
+        """
+        Append records if not already in the table
+        """
+        if obj['id'] not in existing_ids:
+            # only insert if it's not already in the table
+            entries.append(obj)
```

Comment:
Maybe a nitpick, but I don't think you need this function, it's logically pretty simple (not complex enough for an abstraction), what do you think?

Reply:
Good point, saving 2 lines in the loop can be done differently.